site/content/reference/workload_definition/05_op_template_payloads.md

```diff
@@ -326,7 +326,7 @@ ops:
 
 ```
 
-## Binding Points at any position in op field data
+## binding points at any position in op field data
```

Comment:
This file (this section actually) is seeded from the main repo. This change will be overwritten until it is made in the main repo.